docs/old-reference-guide/modules/tuning/pages/rdbms-tuning.adoc

```diff
@@ -0,0 +1,239 @@
+[%collapsible%open]
+====
+- Table `SnapshotEventEntry`, `aggregateIdentifier` column.
+
+- Table `SnapshotEventEntry`, `eventIdentifier` (unique index)
+====
+
+.*Sagas*
+[%collapsible%open]
+====
+- Table `AssociationValueEntry`, columns `sagaType`, `associationKey` and `associationValue`,
+
+- Table `AssociationValueEntry`, columns `sagaId` and `sagaType`,
+====
+
+The default column lengths generated by, for example, Hibernate may work, but won't be optimal.
+A UUID, for example, will always have the same length.
+Instead of a variable length column of 255 characters, you could use a fixed length column of 36 characters for the aggregate identifier.
+
+The `timestamp` column in the `DomainEventEntry` table only stores ISO 8601 timestamps.
+If all times are stored in the UTC timezone, they need a column length of 24 characters.
+If you use another timezone, this may be up to 28. Using variable length columns is generally not necessary, since time stamps always have the same length.
+
+[WARNING]
+====
+It is highly recommended to store all timestamps in UTC format.
+In countries with daylight saving time, storing timestamps in local time may result in sorting errors for events generated around and during the timezone switch.
+This does not occur when UTC is used.
+Some servers are configured to always use UTC.
+Alternatively, you should configure the event store to convert timestamps to UTC before storing them.
+====
+
+The `type` column in the `DomainEventEntry` stores the type identifiers of aggregates.
+Generally, these are the `'simple name'` of the aggregate.
+Even the infamous `AbstractDependencyInjectionSpringContextTests` in Spring only counts 45 characters.
+Here, again, a shorter (but variable) length field should suffice.
+
+== Auto-increment and sequences
+
+[#auto_increment_and_sequences]
+:navtitle: Auto-Increment and Sequences
+
+When using a relational database as an event store, Axon relies on an auto-increment value to allow xref:axon-framework-reference:events:event-processors/streaming.adoc[streaming event processors] to read all events roughly in the order they were inserted.
+We say "roughly", because "insert-order" and "commit-order" are different things.
+
+While auto-increment values are (generally) generated at insert-time, these values only become visible at commit-time.
+This means another process may observe these sequence numbers arriving in a different order.
+While Axon has mechanisms to ensure eventually all events are handled, even when they become visible in a different order, there are limitations and performance aspects to consider.
+
+When a xref:axon-framework-reference:events:event-processors/streaming.adoc[streaming event processor] reads events, it uses the "global sequence" to track its progress.
+When events become available in a different order than they were inserted, Axon will encounter a "gap." Axon will remember these "gaps" to verify that data has become available since the last read.
```

Comment:
I think we should use Axon Framework instead of just Axon. 
```suggestion
When events become available in a different order than they were inserted, Axon Framework will encounter a "gap." Axon Framework will remember these "gaps" to verify that data has become available since the last read.
```